lstchain/scripts/lstchain_merge_hdf5_files.py

```diff
@@ -28,13 +28,22 @@
                     dest='noimage',
                     help='Boolean. True to remove the images',
                     default=False)
-
-args = parser.parse_args()
+                                                                                    
+parser.add_argument('--run-number', '-r', action='store', type=str,                                                           
+                    dest='run_number',                                                                                  
+                    help='Merge files run-wise',                                                                        
+                    default=None)                                                                                       
+                                                                                                                        
+args = parser.parse_args()                                                                                              
 
 
 def main():
 
-    file_list = [args.srcdir + '/' + f for f in os.listdir(args.srcdir) if f.endswith('.h5')]
+    if args.run_number:                                                                                                 
+        file_list = sorted([args.srcdir + '/' + f for f in os.listdir(args.srcdir) if (f.endswith('.h5') and            
```

Comment:
Could you improve the code by using `os.path.join` rather than + '/' at the same time, please?

Reply:
Done